ckanext/dms/templates/snippets/package_list.html

```diff
@@ -0,0 +1,24 @@
+{% ckan_extends %}
+{% block package_list %}
+    <div class="ux ux-sm">
+    <h2 class="row recent_updates_title">Recently updated</h2>
+    <div class="row">
+        {% for dataset in packages %}
+        <div class="col-md-4 col-sm-3 transparent_box">
+            <div class="row"><a href="{{ h.url_for('/dataset/' + dataset.name) }}">{{ dataset.title }}</a></div>
+            {% set hours_modified = h.get_time_diff_in_hours(dataset.resources[0].last_modified) %}
```

Comment:
Don't use `get_time_diff_in_hours`, use the core helper that already exists `h.time_ago_from_timestamp`.